addons/udes_stock/models/stock_picking.py

```diff
@@ -1775,8 +1775,12 @@ def reserve_stock(self):
                     if unsatisfied:
                         if self:
                             # we need to construct our error message before the
-                            # changes are rolled back.
-                            moves = unsatisfied.mapped('move_lines')
+                            # changes are rolled back, and report only products
+                            # that are unreservable.
+                            done = ('done', 'assigned', 'cancelled')
```

Comment:
This is a constant, define it or replace where we use `done` at next line.

Reply:
@ws-mps constant inlined.

Reply:
Thanks!